### PR TITLE
use static message if notification data is not exist

### PR DESCRIPTION
**Pull request by JayChl**
This change is for wedding. 
Before applying new implementation getting notification data by user id, use static message

### CODE REVIEW

File: client/sw.js [hunk 1]
```diff
@@ -1,17 +1,41 @@
 /* eslint-disable */
 
 self.addEventListener('push', function (event) {
-  var data = {};
+
```
Comment (romandev) [inline client/sw.js]: Please remove unnecessary line.

File: client/sw.js [hunk 2]
```diff
@@ -1,17 +1,41 @@
...
+  var title = "";
```
Comment (romandev) [inline client/sw.js]: var title; is enough
or we can declare these variables when it is actually used.

File: client/sw.js [hunk 3]
```diff
@@ -1,17 +1,41 @@
...
+      title, body)
   );
+
 });
 
 self.addEventListener('notificationclick', function (event) {
-});
+  event.notification.close();
 
+  const clickResponsePromise = Promise.resolve();
```
Comment (romandev) [inline client/sw.js]: Why do we need this?
I'm not sure but it looks unnecessary.

File: client/sw.js [hunk 4]
```diff
@@ -1,17 +1,41 @@
 /* eslint-disable */
 
 self.addEventListener('push', function (event) {
-  var data = {};
+
+  var title = "";
+  var body = "";
+
   if (event.data) {
+    var data = {};
```
Comment (romandev) [inline client/sw.js]: Let's merge this line and line11.

File: client/sw.js [hunk 5]
```diff
@@ -1,17 +1,36 @@
...
     self.registration.showNotification(
-      data.notification.title, data.notification)
+      title, body)
   );
+
 });
 
 self.addEventListener('notificationclick', function (event) {
+  event.notification.close();
+  clickResponsePromise = clients.openWindow("https://nadongguri.com/wedding/");
```
Comment (romandev) [inline client/sw.js]: Why do we need a global variable here? (clickResponsePromise)
I didn't find where the global variable is used.

File: client/sw.js [hunk 6]
```diff
@@ -1,17 +1,36 @@
...
+      tag: 'wedding',
+      url: 'https://nadongguri.com/wedding/'
+    };
+    body = notificationOptions;
   }
 
   event.waitUntil(
     self.registration.showNotification(
-      data.notification.title, data.notification)
+      title, body)
```
Comment (romandev) [inline client/sw.js]: We can make these two lines as one line. (and please append semicolon to end of line)
self.registration.showNotification(title, body);

File: client/sw.js [hunk 7]
```diff
@@ -1,17 +1,36 @@
...
   );
+
```
Comment (romandev) [inline client/sw.js]: nit: Please remove unnecessary blank line.

File: client/sw.js (hunk 6 above)
Comment (JayChl) [inline client/sw.js]: event.waitUntil(
    self.registration.showNotification(title, body));
or
  self.registration.showNotification(title, body);  
 without event.waitUntil?

File: client/sw.js (hunk 5 above)
Comment (JayChl) [inline client/sw.js]: This is unnecessary variable will remove this thanks

**Pull request merged**